src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * 40;
+
+  if (days < 3) {
```

Comment:
2. [Checklist] Don't use literals in expressions (HARDCODE). Instead use constants explaining this values
and for the next lines